tests/images_cve_scanner/main.yml

```diff
@@ -0,0 +1,36 @@
+---
+# vim: set ft=ansible:
+#
+# !!!NOTE!!! This playbook was tested using Ansible 2.2; it is recommended
+# that the same version is used.
+#
+# This playbook is a basic security test for container images, please see
+#   the tests/images_cve_scanner/vars.yml for image details.
+#     - Pull container image
+#     - Atomic scan specified container image
+#     - Delete specified container image
+#   Note: please also see roles/container_image_scanner/meta/main.yml, because
+#         the default storage of Atomic Host is very limit, and some container
+#         image are very big, we haven't pull all of container image then scan
+#         them together.
+#
+- name: prepare container images for scanning
+  hosts: all
+  become: yes
+
+  tags:
+    - scan_images
+
+  vars_files:
+    - vars.yml
+
+  tasks:
```

Comment:
Before diving into the test itself, could you include the role that checks the version of Ansible: `ansible_version_check`